5-Heroku/client/src/utils/TodoApi.js

```diff
@@ -1,47 +1,68 @@
 import axios from 'axios'
-const url = `${process.env.TODO_SERVER_URL}/api/todo/`
+const url = '/todos/'
 
 const getTodoById = async (todoId) => {
-    return axios.get(`${url}${todoId}`, {withCredentials: true})
+    return axios.get(`${url}${todoId}`)
     .then(res => {
-        return res.data
+        if (res.status != 200) {
```

Comment:
axios handle non 200 status and throws it to catch.
read about it here:
https://shahata.medium.com/why-i-wont-be-using-fetch-api-in-my-apps-6900e6c6fe78

then plz remove the status check and leave just 
`.then(res => res.data)`